pkg/typesystem/typesystem.go

```diff
@@ -7,72 +7,54 @@ import (
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
-type SchemaVersion int32
-
 const (
-	SchemaVersionUnspecified SchemaVersion = 0
-	SchemaVersion1_0         SchemaVersion = 1
-	SchemaVersion1_1         SchemaVersion = 2
+	SchemaVersion10 = "1.0"
```

Comment:
SchemaVersion # 10? Or 1.0?

I usually see Go separate with underscores in these scenarios. Underscores represent ".", since "." is not an acceptable characters for variable names.

Reply:
GoLand hated it. It said "Use Camel case instead of snake case", but yeah, having the underscore is better. Fixed. Thanks.